src/main/resources/site/pages/default/default.html

```diff
@@ -0,0 +1,91 @@
+    <style type="text/css">.recentcomments a{display:inline !important;padding:0 !important;margin:0 !important;}</style>
+
+    <script data-th-src="${assetUrls.jqueryJs}" src="http://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script type="text/javascript" data-th-src="${assetUrls.superheroJs}" src="../assets/js/superhero.js"></script>
+    <script type="text/javascript" data-th-src="${assetUrls.flexsliderJs}" src="../assets/js/jquery.flexslider-min.js"></script>
+    <meta name="robots" content="noodp" />
+    <meta name="generator" content="Enonic XP" />
+    <meta name="author" content="Enonic" />
+
+    <style data-th-if="${backgroundImageUrl}"
+           data-th-utext="|body.custom-background { background-image: url('${backgroundImageUrl}'); background-repeat: repeat; background-position: top left; background-attachment: scroll;|"
+           type="text/css" id="custom-background-css">
+    </style>
+</head>
+
+<body class="home blog" data-th-class="${bodyClass}" data-portal-component-type="page">
+
+    <div id="page" class="hfeed site" style="padding-top: 250px;">
+        <div id="masthead-wrap" data-th-style="${headerStyle}">
+            <header id="masthead" class="site-header" role="banner">
+
+                <!-- Headings -->
+                <hgroup>
+                    <h1 class="site-title">
+                        <a data-th-href="${siteUrl}"
+                           data-th-title="${title}"
+                           data-th-text="${title}"
+                           href="#"
+                           title="Theme Preview"
+                           rel="home">Theme Preview</a>
+                    </h1>
+                    <h2 class="site-description" data-th-text="${description}">Superhero WordPress theme for Enonic XP</h2>
+                </hgroup>
+
+                <!-- Menu -->
+                <nav data-th-replace="/lib/superhero/fragments::main-navigation">
+                </nav>
+                <div class="clearfix"></div>
+
+            </header>
+        </div>
+
+
+        <!-- Regions -->
+        <div data-th-if="${mainRegion}" data-th-remove="tag">
+            <div data-portal-region="main">
+                <div data-th-each="component : ${mainRegion.components}" data-th-remove="tag">
+                    <div data-portal-component="${component.path}" data-th-remove="tag"/>
+                </div>
+            </div>
+        </div>
+        <div data-th-if="${isFragment}" data-th-remove="tag">
+            <div data-portal-component="fragment" data-th-remove="tag"/>
+        </div>
+
+
+        <!-- Footer -->
+        <div id="colophon-wrap">
+            <footer id="colophon" class="site-footer" role="contentinfo">
+                <div class="site-info" data-th-utext="${footerText}">
+                    Configure footer text.
+                    <!--<a href="http://enonic.com/" title="Fastest way from idea to digital experience." rel="generator">Proudly
```

Comment:
Just remove the old footer text. Or add `<!--*/ /*-->` so thymleaf ignores it in the output

Reply:
Hmm, `Configure footer text.` is an intentional, visible fallback text. The comment thing... I need to find out why that was added. It was even in two places originally, seems awful much for something that has no use?